Automattic/jetpack
# Theme Fields: load is_block_theme only for < WP 6.3

## Proposed changes:

* Adds version check for loading is_block_theme field to wp/v2/themes API response, since this field is added by WP 6.3 and above

See related Core ticket: https://core.trac.wordpress.org/ticket/58123#ticket

**Note:** I'm AFK for quite a while starting next week, so feel free to take over this PR and merge it. Thanks!

### Other information:


## Jetpack product discussion

See previous PR: https://github.com/Automattic/jetpack/pull/30144/files

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* Use REST API Tester from Jetpack debugger to check the response of `wp/v2/themes`. It should include `is_block_theme` from Jetpack if WP version is < 6.3, and from WP when the version is 6.3 or above.
